exercises/perfect-numbers/example.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var Divisors = require('./divisors');
+
+var PerfectNumbers = function() {
+
+}
+
+PerfectNumbers.prototype.classify = function(number) {
+
+  var i, sum, result;
+  var divisors = new Divisors();
+
+  // Check if the input is valid
+  if (number <= 0)
+    return 'Classification is only possible for natural numbers.';
+
+  // Factorize the current number.
+  var divsArray = divisors.getDivisors(number);
+
+  // Sum the factors.
+  sum = 0;
+  for (i = 0; i < divsArray.length; i++) {
+    sum = sum + divsArray[i];
+  }
```

Comment:
Good question.  NodeJS from 4 onward supports arrow functions, and 4 is what we recommend people install for this track.  4 is getting pretty stale, so it makes me think we should bump the version!  The `.yml` file specifies v5.7, so no worries about it building on Travis either.

If you prefer to use ES5 syntax, you can simply write:

```
sum = divsArray.reduce(function(sum, div) { return sum += div}, 0);
```